scania-digital-design-system/sdds-website
# Bump version and content, plus fix for z-index on side-nav

**Describe pull-request**  
_Describe what the pull-request is about_

Bump version
Fix z-index on side-menu, a bit lower
content update

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #

**How to test**  
_Add description how to test if possible_
Start locally or run netlify
Try open a modal, should be covering the whole area
